docs/resources/team_manage.md

```diff
@@ -0,0 +1,46 @@
+# Team Manage Resource
+
+This resource is used to manage teams in Cloudsmith. It allows you to add, update, and remove team members.
+
+## Example Usage
+
+```hcl
+resource "cloudsmith_team_manage" "example" {
+  organization = "example_org"
+  team_name    = "example_team"
+  members {
+      role = "Manager"
+      user = "user1"
+    }
+  members {
+      role = "Member"
+      user = "user2"
+    }
+}
+```
+
+## Argument Reference
+
+> :warning: **Warning**: When running this resource with a **USER API** token (Not applicable to users running this with Service Account API Key) on **newly created teams**, the user will automatically get added to the team causing a `422 error` when specified in the resource as well. We highly advise using service account for this purpose to avoid the error, alternatively, the plan needs to be ran without the user in the resource and re-added after the first apply happens.
```

Comment:
`We highly advise using a service account for this purpose to avoid the error. Alternatively, the plan needs to be run without the user in the resource and re-added after the first apply happens.`